src/components/oscal-utils/OSCALCatalogUtils.js

```diff
@@ -0,0 +1,15 @@
+import matchingProp from "./OSCALPropUtils";
+
+/**
+ * Determines if a control is withdrawn.
+ *
+ * @param {object} control
+ * @returns a boolean describing whether the control is withdrawn
```

Comment:
Can we also update this `@returns`?
```suggestion
 * @returns {boolean} true if control is withdrawn
```